### PR TITLE
Remove uses of zap.SugaredLogger from Prometheus receiver

### PR DESCRIPTION
**Pull request by dinooliva**

### CODE REVIEW

File: receiver/prometheusreceiver/internal/metrics_adjuster.go [hunk 1]
```diff
@@ -350,16 +349,15 @@ func (ma *MetricsAdjuster) adjustPoint(metricType metricspb.MetricDescriptor_Typ
 			&wrappers.DoubleValue{Value: currentSum - initialSum}
 	default:
 		// this shouldn't happen
-		ma.logger.Infof("adjust unexpect point type %v, skipping ...", metricType.String())
+		ma.logger.Info("adjust - unexpect point type, skipping", zap.String("type", metricType.String()))
```
Comment (rghetia) [inline receiver/prometheusreceiver/internal/metrics_adjuster.go]: maybe rephrase to
```go
ma.logger.Info("adjust - skipping unexpected point", zap.String("type", metricType.String()))
```

File: receiver/prometheusreceiver/internal/metricsbuilder.go [hunk 2]
```diff
@@ -98,7 +98,7 @@ func (b *metricBuilder) AddDataPoint(ls labels.Labels, t int64, v float64) error
 				b.scrapeStatus = scrapeStatusOk
 			} else {
 				b.scrapeStatus = scrapeStatusErr
-				b.logger.Warnw("http client error:", "ts", t, "value", v, "labels", lm)
+				b.logger.Warn("http client error", zap.Int64("timestamp", t), zap.Float64("value", v), zap.String("labels", fmt.Sprintf("%v", lm)))
```
Comment (owais) [inline receiver/prometheusreceiver/internal/metricsbuilder.go]: Zap has a Time field that could possible b used. https://godoc.org/go.uber.org/zap#Time

File: receiver/prometheusreceiver/internal/metrics_adjuster.go (hunk 1 above)
Comment (dinooliva) [reply receiver/prometheusreceiver/internal/metrics_adjuster.go]: Done.

File: receiver/prometheusreceiver/internal/metricsbuilder.go (hunk 2 above)
Comment (dinooliva) [reply receiver/prometheusreceiver/internal/metricsbuilder.go]: I'd prefer leave it as the raw data as received rather than convert it into time.Time but I can do the conversion if you prefer.

Comment (owais) [reply receiver/prometheusreceiver/internal/metricsbuilder.go]: I overlooked it and thought it was a time.Time value. Raw definitely makes sense.
